core/mount/mount_idmapped_linux.go

```diff
@@ -61,14 +61,14 @@ func parseIDMapping(mapping string) ([]syscall.SysProcIDMap, error) {
 	}, nil
 }
 
-// IDMapMount applies GID/UID shift according to gidmap/uidmap for target path
```

Comment:
The interface is now stable, add a function with the new signature. This one can be marked as deprecated if it should never be used with the default values.